packages/auto-drive/tests/chunker.spec.ts

```diff
@@ -1,17 +1,26 @@
-import { createNode } from '@ipld/dag-pb'
-import { cidOfNode } from '../src'
-import { createFileIPLDDag, createFolderIPLDDag, createMetadataIPLDDag } from '../src/ipld/chunker'
+import { createNode, decode, PBNode } from '@ipld/dag-pb'
```

Comment:
I see in your test you still use decode and other function from @ipld/* dependencies, could we expose the functions a user will need from our package, so they can import everything they need, directly from our package, and apply the same behavior in our test?